pynest/examples/balancedneuron.py

```diff
@@ -65,16 +65,16 @@
 
 
 t_sim = 25000.0  # how long we simulate
-n_ex = 16000     # size of the excitatory population
-n_in = 4000      # size of the inhibitory population
-r_ex = 5.0       # mean rate of the excitatory population
-r_in = 20.5      # initial rate of the inhibitory population
-epsc = 45.0      # peak amplitude of excitatory synaptic currents
-ipsc = -45.0     # peak amplitude of inhibitory synaptic currents
-d = 1.0          # synaptic delay
-lower = 15.0     # lower bound of the search interval
-upper = 25.0     # upper bound of the search interval
-prec = 0.01      # how close need the excitatory rates be
+n_ex = 16000  # size of the excitatory population
+n_in = 4000  # size of the inhibitory population
+r_ex = 5.0  # mean rate of the excitatory population
+r_in = 20.5  # initial rate of the inhibitory population
+epsc = 45.0  # peak amplitude of excitatory synaptic currents
+ipsc = -45.0  # peak amplitude of inhibitory synaptic currents
+d = 1.0  # synaptic delay
+lower = 15.0  # lower bound of the search interval
+upper = 25.0  # upper bound of the search interval
+prec = 0.01  # how close need the excitatory rates be
```

Comment:
In places like this, I think readbility is better if the comments are lined up. Is that feasible?

Reply:
Yes, we can use `# fmt: off` and `# fmt: on` blocks

Reply:
@heplesser How can we go about identifying these locations?

Reply:
I think we would need to build a little parser that can identify subsequent lines in which `#` are
- aligned
- at least one `#` is more than two spaces from preceding code
- not sure if we have any cases with just a single space in front of the `#`, if so we would need to handle that, too.
Not sure if one can express that as a regular expression.
Then run it on the pre-black code.

Reply:
I think in this case a Python script might be much easier to make than a regular expression, especially given the one off nature of the task :) I'll get on that.

Reply:
Just a suggestion. We could also denote comments that should be aligned by `#:`, that is,

```
 n_ex = 16000  #: size of the excitatory population
 n_in = 4000   #: size of the inhibitory population
 r_ex = 5.0    #: mean rate of the excitatory population
```

This will most likely make parser script easier and less prone for making unwanted alignments elsewhere.

Reply:
This would be a convention introduced by us, and as far as I know not something that can be utilized by black. I think we have to manually add `#:` to the blocks where we want comments to be aligned for readability. These will predominately be located in example scripts, as the one @heplesser referred to above. 

Ideally, I think we want a parser script that runs after black and aligns the comment blocks with `#:`. The example scripts are prone to changes and we want the alignments to be continuously updated in case we, for instance, change the length of a variable name. 

If we want to go with this solution, it could also probably be a separate PR. @heplesser Thoughts?

Reply:
Usually scripts that modify the source code from `push`-CI is discouraged, because it adds more commits and CI-runs, and because we'd have to make and maintain a GitHub API integrations (the runners have no git authentication with write-access to the repo, iirc, and by design).

If we want such a script it might be an idea to add it to the same `pre-commit` tool that runs the black pre-commit hook locally on a dev's machine before it is even pushed to the remote and any CI check runs. Then, in CI, we can run a simple check and reject the CI if it contains any misaligned `#:`. This would be completely analogous to how `black` is used in the repo.

Reply:
Isn't it possible to put a `#noqa` tag around such blocks?